examples/hotrod/pkg/tracing/rpcmetrics/observer_test.go

```diff
@@ -47,46 +48,39 @@ func withTestTracer(runTest func(tt *testTracer)) {
 			semconv.ServiceNameKey.String("test"),
 		)),
 	)
-	tracer, _ := otbridge.NewTracerPair(tp.Tracer(""))
 	runTest(&testTracer{
 		metrics: metrics,
-		tracer:  tracer,
+		tracer:  tp,
 	})
 }
 
 func TestObserver(t *testing.T) {
 	withTestTracer(func(testTracer *testTracer) {
 		ts := time.Now()
-		finishOptions := opentracing.FinishOptions{
-			FinishTime: ts.Add(50 * time.Millisecond),
-		}
+		finishOptions := trace.WithTimestamp(ts.Add((50 * time.Millisecond)))
 
 		testCases := []struct {
 			name           string
-			tag            opentracing.Tag
+			tag            trace.SpanKind
 			opNameOverride string
 			err            bool
 		}{
-			{name: "local-span", tag: opentracing.Tag{Key: "x", Value: "y"}},
```

Comment:
in OT span kind is an example of tag, but this test was also using another tag, unrelated to span kind. Your change removes that - did not not have any impact on the tests?

Reply:
Sorry Yuri. How would I replace it with otel traces spankind? Do I need to use attribute keys?

Reply:
span kind can be a separate field in the test case, and tag can remain as arbitrary tag (decoupled from span kind)

Reply:
arbitrary tag means?